### PR TITLE
Specify supported SET_ATTITUDE_TARGET message for offboard mode

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
This PR specifies in more detail SET_ATTITUDE_TARGET mavlink message supported for offboard mode.

This PR is a follow up from #545

### CODE REVIEW

File: en/flight_modes/offboard.md [hunk 1]
```diff
@@ -38,11 +38,15 @@ The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL
...
+  * Attitude/orientation with thrust setpoint
+  * Body rate with thrust setpoint
 
 ### Fixed Wing
 
 * [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET) - Control vehicle attitude/body rates.
+  * Attitude/orientation with thrust setpoint
+  * Body rate  with thrust setpoint
 
 > **Warning** Position setpoints are **not supported** on Fixed Wing offboard mode (`SET_POSITION_TARGET_LOCAL_NED`).
```
Comment (hamishwillee) [inline en/flight_modes/offboard.md]: @Jaeyoung-Lim Is this still ture?

File: en/flight_modes/offboard.md [hunk 2]
```diff
@@ -38,11 +38,15 @@ The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL
       > **Note** Acceleration setpoint values are mapped to create a normalized thrust setpoint (i.e. acceleration setpoints are not "properly" supported).
     * Position setpoint **and** velocity setpoint (the velocity setpoint is used as feedforward; it is added to the output of the position controller and the result is used as the input to the velocity controller).
   - PX4 supports the coordinate frames (`coordinate_frame` field): [MAV_FRAME_LOCAL_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_NED) and [MAV_FRAME_BODY_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_BODY_NED).
-* [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET) - Control vehicle attitude/orientation.
+* [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET)
```
Comment (hamishwillee) [inline en/flight_modes/offboard.md]: Thanks @Jaeyoung-Lim - So just to confirm what this means -

* Attitude/orientation with thrust setpoint - this means you can set `SET_ATTITUDE_TARGET.thrust` and `SET_ATTITUDE_TARGET.q`
* Body rate  with thrust setpoint - means you can set `body_roll_rate` ,`body_pitch_rate`, `body_yaw_rate` and `thrust` at the same time?

Comment (Jaeyoung-Lim) [reply en/flight_modes/offboard.md]: Yes, exactly

File: en/flight_modes/offboard.md (hunk 1 above)
Comment (Jaeyoung-Lim) [reply en/flight_modes/offboard.md]: Yes, the PR has requested a test flight. Once the position setpoint gets merged, I will update the documentation accordingly!

**Comment by hamishwillee** [comment]
Awesome thanks @Jaeyoung-Lim ! I merged and [copied this in explicitly](https://github.com/PX4/px4_user_guide/commit/a31edac721ffe0ce0b9559d7f1a465a8136848b7)

**Pull request merged**